source/pins_arduino.h

```diff
@@ -75,7 +75,11 @@ const static uint8_t SCL1 = 5;
 const static uint8_t SDA1 = 3;
 const static uint8_t CS0 = 24;
 const static uint8_t CS1 = 26;
+const static uint8_t CE0 = 24;
+const static uint8_t CE1 = 26;
+static const uint8_t SS = 24;
```

Comment:
Good catch! I've fixed this one. x86/x64 now include the same declarations as RPI.
For readability, I've left the assignments inside the `#ifdef` for each.